sycl/test/matrix/matrix-bfloat16-test.cpp

```diff
@@ -0,0 +1,192 @@
+// RUN: %clangxx -fsycl -O2 %s -o %t.out
+#include <CL/sycl.hpp>
```

Comment:
```suggestion
#include <sycl/sycl.hpp>
```

Reply:
it is "include/sycl/CL/sycl.hpp", how could we change into "#include <sycl/sycl.hpp>"